src/scss/_packages.scss

```diff
@@ -2,80 +2,172 @@
 @import "mixins";
 
 .r-tiles {
-    @apply items-center;
-
     @screen md {
-        @apply flex;
+        @apply grid grid-cols-2 grid-flow-row-dense gap-8;
     }
 
-    .r-tile {
-        @apply rounded shadow border border-gray-300 cursor-pointer p-4 font-display;
+    .package {
 
-        width: 525px;
-        max-width: 525px;
+        .r-tile {
+            @apply rounded-xl shadow border border-gray-300 p-4 font-display;
+            @apply flex flex-col;
 
-        .logo {
-            @apply mr-4;
+            width: 100%;
 
-            width: 48px;
-            height: 48px
-        }
+            .r-tile-header {
+                @apply flex flex-row items-center mb-4;
+
+                .logo {
+                    max-width: 20%;
+                    height: 32px;
+                }
 
-        &.featured {
-            @apply cursor-default;
+                .header-link {
+                    @apply text-blue-600 rounded;
 
-            max-height: 350px;
-            height: 350px;
+                    padding: 8px 16px 8px 16px;
+
+                    &:hover {
+                        @apply text-blue-700;
+
+                        box-shadow: 0 4px 4px rgba(0, 0, 0, 0.2), 0 4px 4px rgba(0, 0, 0, 0.14), 0 1px 5px rgba(0, 0, 0, 0.12);
```

Comment:
Why is this box shadow "special"? We should use our already-defined utility classes for these unless there's some reason we can't.

Reply:
If we have one standard for buttons in our site, I have no concerns using that. I didn't see one other than the `.btn` classes that were highly specific to the buttons on the home page of pulumi.com.